src/Tablebot/Handler.hs

```diff
@@ -99,3 +113,33 @@ runCron pool (CCronJob delay fn) = do
 -- | @killCron@ takes a list of @ThreadId@ and kills each thread.
 killCron :: [ThreadId] -> IO ()
 killCron = mapM_ killThread
+
+-- | Given a list of compiled application commands and a pointer to the
+-- tablebot cache, create the given application commands, purge ones that
+-- weren't created by us, and place the application command id's and their
+-- actions in the cache.
+submitApplicationCommands :: [CompiledApplicationCommand] -> MVar TablebotCache -> DiscordHandler ()
+submitApplicationCommands compiledAppComms cacheMVar =
+  ( do
+      -- generate the application commands, cleaning up any application commands we don't like
+      serverIdStr' <- liftIO $ lookupEnv "SERVER_ID"
+      case serverIdStr' of
+        Nothing -> pure ()
```

Comment:
Is this correct? I thought a blank string lead to global application commands or something?

Reply:
if the environment variable is not present, application commands will not be generated. if the variable is present but is `"global"`, global commands will be created. else, the specific named server will be used for the application commands.